tests/nlu/test_train.py

```diff
@@ -15,7 +15,7 @@ def as_pipeline(*components):
 
 
 def pipelines_for_tests() -> List[Tuple[Text, List[Dict[Text, Any]]]]:
-    # these templates really are just for testing
+    # these responses really are just for testing
```

Comment:
I think this should be `templates`

Reply:
I think it should be `pipelines`. There used to be `pipeline templates` (you can use `template` wherever if you want to 😂 ) which were shortcuts for a `pipeline`. You could then use something like `supervised_embeddings` instead of having to define all the single NLU components.  That's the ancient docs entry where you can see it in action: https://legacy-docs.rasa.com/docs/nlu/0.15.1/choosing_pipeline/